README.md

```diff
@@ -12,4 +12,26 @@
 | $$                 \$$    $$                                                         \$$    $$          
  \$$                  \$$$$$$                                                           \$$$$$$             
 ```
-#### Javascript library for working with the Payment Contracts in ArGo
+#### Typescript library for working with the Payment Contracts in ArGo
+
+> Important Notice: ArGo is in its Alpha stage. If you have a suggestion, idea, or find a bug, please report it! The ArGo team will not be held accountable for any funds lost.
+
+## About ArGo
+ArGo is a simple & powerful platform to deploy your frontend app to Arweave Permaweb where your app will live for eternity.
+
+## Build argo-payments-js 
+Follow below steps to build argo-payments-js in local
+
+> Note: Our current latest stable branch is Master. After clone make sure you are on master branch
+ - Clone argo-payments-js code into your local by using `git clone https://github.com/argoapp-live/argo-payments-js.git`
+ - Install packages using either yarn or npm from root folder `yarn` or `npm install`
+ - Run `yarn build` or `npm run dev`
+
+
+## Run argo-payments-js tests
+ - Clone argo-payments-js code into your local by using `git clone https://github.com/argoapp-live/argo-payments-js.git`
+ - Install packages using either yarn or npm from root folder `yarn` or `npm install`
+ - Make sure you add all the values in `.env` file before running tests.
```

Comment:
Do we need `.env` to run the tests

Reply:
If so, what does it look like?

Reply:
yes we need a few env variables
```COINMARKETCAP_KEY=""
BICONOMY_KEY=""
MNEMONIC_TEST=""
RPC_ENDPOINT=""```
I will put it in README